.github/workflows/ci.yml

```diff
@@ -11,18 +11,13 @@ on:
 jobs:
   build:
     runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        node: [16]
```

Comment:
Are we all good by just mentioning the node version here without setting up that version? Does that happen by itself when we specify this?

Reply:
Oops! it was my mistake. Updated now